README.md

```diff
@@ -155,6 +170,40 @@ q.exact("rs2032651", "D"); // will only match genotype D
 q.exact("rs2032651", "AT"); // will only match genotype AT
 ```
 
+#### export()
```

Comment:
@finnp toJSON is expected to return an object with no hidden properties (getters/setters/whatnot) i think functions are fine to return from that. what JSON.stringify does with it will strip the functions out